website/docs/reference/commands/build.md

```diff
@@ -77,3 +78,15 @@ Completed successfully
 
 Done. PASS=7 WARN=0 ERROR=0 SKIP=0 TOTAL=7
 ```
+
+## Functions
+_Available from dbt Core v1.11 and in the <Constant name="fusion_engine" />_
+
+The `build` command runs [user-defined functions](/docs/build/udfs) as part of the DAG execution. To build or rebuild only `functions` in your project, run `dbt build --select "resource_type:function"`. For example:
```

Comment:
again, can we say the build command "builds" user-defined functions - run is just for models